### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_ColorContrast: The drop down color contrasts is below 3:1

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#11196]
["Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11yTCS;#A11yWCAG2.1;#Benchmark;#DesktopApp;#WCAG1.4.11;#SQL Azure Data Studio;#FTP;#ColorContrast;#A11ySev2;#Acc123;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.20.0-insider (system setup)
Commit: fd29e35
Date: 2020-07-02T05:40:02.292Z
VS Code: 1.46.0
Electron: 7.3.2
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:WCAG1.4.11
Tool: Accessibility insights for desktop

**Repro Steps:**
1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Right-click connection and click New Notebook4. Open the Command Palette by pressing CTRL+SHIFT+P (Windows) 
 appears.
4. Type “Python” into the text box that appears. Select “Configure Python for Notebooks” from the menu.
Python Dependencies Wizard will appear.
5. Provide any Input file details & Next.
6.  Launch Accessibility insights for desktop and select third tab contrast.

**Actual:**
Issue1:
When the color contrast of the dropdown is checked the ratio is 2.847:1

Issue2:
The color contrast of the selected item to the item, not selected  is 1.221:1

**Expected:**
Issue1:
The color contrast of the dropdown should be 3:1 or greater.

Issue2:
The color contrast of the  selected item to the item, not selected should be 3:1 or greater

**User Impact:**
Low vision users will face difficulty in viewing the separate sections in the lsit .

**Attachment link for Reference:**
![#11196A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_ColorContrast](https://user-images.githubusercontent.com/24580570/86462863-fcd06700-bd49-11ea-843f-be98306204e9.png)
![#11196A11y_AzureDataStudioStayHealthy_Notebook_Python Dependencies Wizard_ColorContrast2](https://user-images.githubusercontent.com/24580570/86462945-2c7f6f00-bd4a-11ea-95d1-ce41d3b77424.png)

**Comment by alanrenmsft** [comment]
@fsteffi we are using the same dropdown control as vscode, see screenshot below, looks like they also have the same problem. please open an issue in vscode repository, once they fix that we will also get the fix.

![image](https://user-images.githubusercontent.com/13777222/92276686-5538f780-eea6-11ea-8c48-ed0c5d236609.png)

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro
Version 1.42.0-insider(user-setup)
![11196_issue1 fixed](https://user-images.githubusercontent.com/93374043/218382343-d7af8c42-7105-4f0a-b5fa-106a70f6f1b4.png)
![11196_issue2 fixed](https://user-images.githubusercontent.com/93374043/218382358-8f847179-0b20-4d9d-8c4a-418fdee9ed65.png)

**GithubTags**:#Closed;
